contracts/src/Teleporter/upgrades/TeleporterUpgradeable.sol

```diff
@@ -5,7 +5,7 @@
 
 pragma solidity 0.8.18;
 
-import "./TeleporterRegistry.sol";
+import {TeleporterRegistry} from "./TeleporterRegistry.sol";
 
 /**
  * @dev TeleporterUpgradeable provides upgrade utility for applications built on top
```

Comment:
can't comment on line, but confirming for `minTeleporterVersion` we can't change visbility, since derived contracts will need to implement `updateMinTeleporterVersion` and have access to the state variable. Trying to think how we can only allow updating in `updateMinTeleporterVersion`

Reply:
Perhaps we should add a comment at that state variable explaining this special case and why it diverges from Solidity best practices.

Reply:
Good call. Adding now